features/step_definitions/book_index_page_steps.rb

```diff
@@ -28,3 +28,15 @@
 Then(/^I am shown a list of books with that title$/) do
   expect(page).to have_content(Book.find(1).title)
 end
+
+Given(/^some books have been ordered$/) do
+  pending # Write code here that turns the phrase above into concrete actions
+end
+
+When(/^I sort by "([^"]*)"$/) do |sort|
```

Comment:
Unused block argument - `sort`. You can omit the argument if you don't care about it.